3-With-Intune MAM/IntuneClient/active-directory-xamarin-intune/App.xaml.cs

```diff
@@ -9,10 +9,16 @@ public partial class App : Application
         public App()
         {
             InitializeComponent();
-
+            AppDomain.CurrentDomain.UnhandledException += CurrentDomain_UnhandledException;
             MainPage = new MainPage();
         }
 
+        private void CurrentDomain_UnhandledException(object sender, UnhandledExceptionEventArgs e)
+        {
+            System.Diagnostics.Debug.WriteLine($"Unhandled Exception {e.ExceptionObject?.ToString()}");
```

Comment:
why would we want customers to have this here? Is it actionable? Debugger.Break is not actionable.

Reply:
When unhandled exception is thrown, it will attach the debugger or if already attached, this line will act as breakpoint. If app closes unexpectedly, this will tell whether it was caused by an exception in the app or was it closed externally (such as manual or by Intune)